test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/HelpInfoTests.cs

```diff
@@ -49,6 +49,38 @@ public async Task Help_WhenNoExtensionRegistered_OutputDefaultHelpContent(string
         testHostResult.AssertOutputMatches(wildcardMatchPattern);
     }
 
+    [ArgumentsProvider(nameof(TargetFrameworks.All), typeof(TargetFrameworks))]
+    public async Task HelpShortName1_WhenNoExtensionRegistered_OutputDefaultHelpContent(string tfm)
+    {
+        var testHost = TestInfrastructure.TestHost.LocateFrom(_testAssetFixture.NoExtensionTargetAssetPath, TestAssetFixture.NoExtensionAssetName, tfm);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--h");
+
+        testHostResult.AssertExitCodeIs(ExitCodes.Success);
+
+        const string wildcardMatchPattern = $"""
+.NET Testing Platform v*
+Usage {TestAssetFixture.NoExtensionAssetName}* [option providers] [extension option providers]
```

Comment:
it might be better to shorten this output so we are sure this is help, but don't have to fix the exact wording of all the options in 10 places that are testing a single param.

Reply:
Let's postpone this for now.